docs/ops/pooling/AvgPool_1.md

```diff
@@ -68,26 +69,87 @@
     * *same_upper (same_lower)* the input is padded to match the output size. In case of odd padding value an extra padding is added at the end (at the beginning).
     * *valid* - do not use padding.
   * **Type**: string
-  * **Default value**: None
+  * **Default value**: *explicit*
   * **Required**: *no*
   * **Note**: *pads_begin* and *pads_end* attributes are ignored when *auto_pad* is specified.
 
 **Inputs**:
 
-*   **1**: 4D or 5D input tensor. Required.
+*   **1**: 3D, 4D or 5D input tensor. Required.
+
+**Outputs**:
+  * **1**: Input shape can be either (N,C,H), (N,C,H,W) or (N,C,H,W,D). Then the corressponding output shape will be (N,C,H_out), (N,C,H_out,W_out) or (N,C,H_out,W_out,D_out)
+       
+        H_out = (H + pads_begin[0] + pads_end[0] - kernel[0] / strides[0]) + 1
+        
+        W_out = (H + pads_begin[1] + pads_end[1] - kernel[1] / strides[1]) + 1
+        
+        D_out = (H + pads_begin[2] + pads_end[2] - kernel[2] / strides[2]) + 1
 
 **Mathematical Formulation**
 
 \f[
 output_{j} = \frac{\sum_{i = 0}^{n}x_{i}}{n}
 \f]
 
-**Example**
+**Examples**
 
 ```xml
 <layer ... type="AvgPool" ... >
```

Comment:
All cases are for `exclude_pad="true"`. Please, provide examples where `exclude_pad="false"`